src/platform/unix/mod.rs

```diff
@@ -660,10 +658,68 @@ fn make_socket_lingering(sockfd: c_int) -> Result<(),UnixError> {
     Ok(())
 }
 
+struct BackingStore {
+    fd: c_int
+}
+
+impl BackingStore {
+    pub fn new(length: usize) -> BackingStore {
+        let count = GLOBAL_SHM_COUNT.fetch_add(1, Ordering::Relaxed);
+        let timestamp = UNIX_EPOCH.elapsed().unwrap();
+        let pid = unsafe { libc::getpid() };
```

Comment:
Sorry, you mentioned this earlier but I didn't get around to it. Updated per the above comment.